WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ShippingLabelPaperSizeOptionListView.swift

```diff
@@ -0,0 +1,51 @@
+import Networking
+import SwiftUI
+
+/// Displays a grid view of all available paper size options for printing a shipping label.
+struct ShippingLabelPaperSizeOptionListView: View {
+    private let paperSizeOptions: [ShippingLabelPaperSize]
+    private let numberOfColumnsPerRow = 2
+    private let numberOfRows: Int
+
+    init(paperSizeOptions: [ShippingLabelPaperSize]) {
+        self.paperSizeOptions = paperSizeOptions
+        self.numberOfRows = Int(ceil(Double(paperSizeOptions.count) / Double(numberOfColumnsPerRow)))
+    }
+
+    var body: some View {
+        ScrollView {
```

Comment:
Not today: I wonder if we could also make a separate `WooScrollView` component which is already configured with the standard background that we use.

Reply:
Great idea! I created a separate ticket to document SwiftUI adoption and create custom components in https://github.com/woocommerce/woocommerce-ios/issues/3379 for future improvements 😃